.github/copilot-instructions.md

```
@@ -0,0 +1,87 @@
+|------|----------|
+| `api/v1alpha1/` | CRD type definitions and webhook validation |
+| `cmd/` | Binary entrypoints (operator and vgmanager subcommands) |
+| `internal/controllers/` | Reconcilers: lvmcluster, vgmanager, persistent-volume, node removal |
+| `internal/controllers/lvmcluster/resource/` | Resource managers (DaemonSet, StorageClass, CSIDriver, etc.) |
+| `config/` | Kustomize overlays, CRD manifests, RBAC, samples |
+| `test/e2e/` | End-to-end test suite |
+| `docs/` | Architecture, design, conventions, ADRs |
+
+## Commit Conventions
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)